src/boot/boot.c

```diff
@@ -1791,6 +1796,86 @@ static void generate_boot_entry_titles(Config *config) {
         }
 }
 
+typedef struct {
+        size_t start_index;
+        size_t end_index;
+} Indices;
```

Comment:
This is really strange.

Reply:
Why?
I need a way to track the start and end indices of entry groups (see below). The start and end index of a group always belong together and are "worthless" without the other, so my OOP patterns kicked in to create the struct to also always save the start and end index of a group in the same structure. :)
Is this not "the C way"?